core-java-modules/core-java-concurrency-advanced-4/src/main/java/com/baeldung/producerconsumer/Producer.java

```diff
@@ -41,7 +45,7 @@ private Message generateMessage() {
         System.out.printf("[%s] Generated Message. Id: %d, Data: %f\n", Thread.currentThread().getName(), message.getId(), message.getData());
 
         //Sleeping on random time to make it realistic
-        ThreadUtil.sleep((long) (message.getData() * 100));
+        // ThreadUtil.sleep((long) (message.getData() * 100));
```

Comment:
Rollback any changes used for testing requester's claim

Reply:
Done